docs: mark deprecated options as such (#13989)
sveltejs/svelte: documentation/docs/05-special-elements/06-svelte-options.md

The `<svelte:options>` element provides a place to specify per-component compiler options, which are detailed in the [compiler section](svelte-compiler#compile). The possible options are:

- `runes={true}` — forces a component into _runes mode_ (see the [Legacy APIs](legacy-overview) section)
- `runes={false}` — forces a component into _legacy mode_

> - `namespace="..."` — the namespace where this component will be used, most commonly "svg"; use the "foreign" namespace to opt out of case-insensitive attribute names and HTML-specific warnings

Reviewer: `foreign` doesn't exist anymore, does it?

Author: ```suggestion
- `namespace="..."` — the namespace where this component will be used, can be "html" (the default), "svg" or "mathml"
```
